packages/server-core/src/projects/project-permission/project-permission.hooks.ts

```diff
@@ -218,7 +219,12 @@ export default {
       () => schemaHooks.validateQuery(projectPermissionQueryValidator),
       schemaHooks.resolveQuery(projectPermissionQueryResolver)
     ],
-    find: [enableClientPagination(), iff(checkUserScopes, checkPermissionStatus)],
+    find: [
+      enableClientPagination(),
+      iff(checkUserScopes, checkPermissionStatus),
+      resolveProjectId(),
+      discardQuery('project')
```

Comment:
If we have project id in studio then we dont need this overhead of passing project name and then getting id from it.

Otherwise it looks good.

Reply:
No we don't, I had already explained this, we only use project name in studio. 
![image](https://github.com/EtherealEngine/etherealengine/assets/33644205/e3c8bd79-2e75-40a3-99ff-5bb2147b09a6)